mod-finance/examples/fund.sample

```diff
@@ -23,9 +23,13 @@
     "077274ad-6b4f-4c28-9779-6d381e7a1ca1"
   ],
   "restrictByLocations": true,
-  "locationIds": [
-    "eb2d063a-5b4c-4cab-8db1-5fc5c5941df6",
-    "fcd64ce1-6995-48f0-840e-89ffa2288371"
+  "locations": [
+    {
+      locationId: "eb2d063a-5b4c-4cab-8db1-5fc5c5941df6"
+    },
+    {
+      locationId: : "fcd64ce1-6995-48f0-840e-89ffa2288371"
```

Comment:
two occurences of :
And the same also in other places

Reply:
removed